themes/openaq/layouts/partials/footer.html

```diff
@@ -3,9 +3,7 @@
     <section class="nav-section">
```

Comment:
Any stylistic changes in here just automatically come from my prettier config, I am unopinionated so just let me know if you want me to remove 😄